test/test-remote/tsconfig.json

```diff
@@ -28,5 +28,9 @@
     "loki-node-client-tools/*.ts",
     "prom-node-client-tools/resources/*.json",
     "prom-node-client-tools/*.ts"
+  ],
+  "references": [
+    { "path": "../../lib/utils" },
+    { "path": "../../lib/kubernetes" }
```

Comment:
I see! : ) that's how things are not independent anymore. Yeah, that was inevitable and makes sense in the long run I guess.